doc/src/odin/index_markdown.md

```diff
@@ -0,0 +1,451 @@
+Odin II
+=======
+
+Odin II is used for logic synthesis and elaboration, converting a subset
+of the Verilog Hardware Description Language (HDL) into a BLIF netlist.
+
+INSTALL
+-------
+
+### Prerequisites
+
+1.  ctags
+2.  bison
+3.  flex
+4.  gcc 5.x
+5.  cmake 2.8.12 (minimum version)
+6.  time
+7.  cairo
+
+### Build
+
+To build you may use the Makefile wrapper in the ODIN\_ROOT `make build`
+To build with debug symbols you may use the Makefile wrapper in the
+ODIN\_ROOT `make debug`
+
+
+>_____
+>
+> **NOTE**
+>
+> ODIN uses CMake as it's build system. CMake provides a portable cross-platform build systems with many useful features.
+> :   For unix-like systems we provide a wrapper Makefile which supports
+>     the traditional make and make clean commands, but calls CMake
+>     behind the scenes.
+>
+> **WARNING**
+>
+> After you build Odin, please run from the ODIN\_ROOT `make test`. This will simulate and verify all of the included microbenchmark
+circuits to ensure that Odin is working correctly on your system.
+>  
+>   _____
+
+
+
+
+USAGE
+-----
+
+./odin\_II [args]
+
+### Required [args]
+|  arg  | following argument      | Description                                            |
+|-------|:-----------------------:|------------------------------------------------------- |
+| `-c`  |  XML Configuration File |fpga_architecture_file.xml format is specified from VPR |
+|  `-V`|  Verilog HDL FIle       |You may specify multiple verilog HDL files for synthesis|
+|  `-b` |  BLIF File              |                                                        |
+
+<A little misleading>
+
```

Comment:
For this I found it misleading because of run odin with only one of these arguments, could I add a paragraph that these are the arguments necessary to get the full use of odin

Reply:
yeah! I don't know that I care for it either
go for it